content/software/ide-v2/tutorials/getting-started/02.ide-v2-uploading-a-sketch/ide-v2-uploading-a-sketch.md

```diff
@@ -52,3 +52,34 @@ Uploading a sketch is quick and easy, but let's take a look at what we need to d
 
 Congratulations, you have now uploaded a sketch to your Arduino board!
 
+### Please Read (Only Linux Users)
+
+It might happen that when you upload a sketch - after you have selected your board and the serial port -, you get an error **"Error opening serial port ..."**. If you get this error, you need to set serial port permission.

```

Comment:
```suggestion
It might happen that when you upload a sketch - after you have selected your board and the serial port -, you get an error such as **"... Permission denied"**. If you get this error, you need to set serial port permission.
```

Although the different uploading tools of the different boards produce different error messages in this situation, even after quite a bit of testing I still didn't find a single board that produced an error message resembling **"Error opening serial port ..."**. Maybe the error message had that format back in 2008 when this content was written, but I don't find any evidence for it now.

You can see a random example of the common error message here:

https://forum.arduino.cc/t/permission-denied-on-dev-ttyacm0/475568

```text
avrdude: ser_open(): can't open device "/dev/ttyACM0": Permission denied
```